Doc/library/collections.abc.rst

```diff
@@ -282,7 +283,7 @@ Collections Abstract Base Classes -- Detailed Descriptions
 .. class:: Set
            MutableSet
 
-   ABCs for read-only and mutable sets.
+   ABCs for read-only and mutable :ref:`sets <types-set>`.
```

Comment:
I would leave this one without markup.  We're talking in about plain English sets rather than a specific set type.